src/public/js/util.js

```diff
@@ -38,6 +38,12 @@ var UTIL = (function() {
      0xb4, 0x7c, 0xb8, 0x26, 0x77, 0x99, 0xe3, 0xa5,
      0x67, 0x4a, 0xed, 0xde, 0xc5, 0x31, 0xfe, 0x18,
      0x0d, 0x63, 0x8c, 0x80, 0xc0, 0xf7, 0x70, 0x07]);
+  log3.get = function(x) {
```

Comment:
Actually something neat about typed arrays too that you may or may not care about :P 

Arrays are simply an object and the array itself is more like a pointer map. so Arr[0] has a pointer to element 0 of Arr and Arr[1] has a pointer to element 1 etc.

Typed Arrays work quite differently. They work like arrays in most languages where the array is contiguously stored in memory and the array data just points to the first element and then you have a length to tell you where it stops. This is why typed arrays have no .push() method. It also means typed arrays are more efficient in memory usage and performance. Especially if you have an array like ours that can likely all be loaded in to 1 block of cache(<- although that is getting really nitty gritty on performance :D )